.markdownlint-cli2.cjs

```diff
@@ -24,5 +31,11 @@ module.exports = {
     straightQuotes: true,
   },
   customRules: [straightQuotes],
-  ignores: ['**/node_modules/**', '**/*-zh.md', '**/*-pt.md', '.github/PULL_REQUEST_TEMPLATE.md'],
+  ignores: [
+    '**/node_modules/**',
+    '**/*-zh.md',
+    '**/*-pt.md',
+    '**/build/**',
```

Comment:
We were missing this, for those that do releases locally, they should have a couple of changelogs in the build that are fail the lining rule.